src/hooks/useBooleanTimeout.js

```diff
@@ -0,0 +1,20 @@
+import { useState, useEffect } from "react";
+
+const useBooleanTimeout = (timeout = 1000) => {
```

Comment:
did you write this or copy from somewhere? I like adding a link in for copied code so you can refer back to the source, see potential updates, etc.

Reply:
Nope, I wrote this myself!